hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/UpsertPartitioner.java

```diff
@@ -90,8 +94,11 @@ public class UpsertPartitioner<T> extends SparkHoodiePartitioner<T> {
   public UpsertPartitioner(WorkloadProfile profile, HoodieEngineContext context, HoodieTable table,
```

Comment:
Should we add the implementation to a new class, may be sortedUpsertPartitioner or something, so there is a clean separation. We can use the same config to control which one gets called.

Reply:
Done. The changes were minimal, hence did not add it earlier.